### PR TITLE
Update get-test-results.sh

### PR DESCRIPTION
**Pull request by assoomro**
[AB#7961](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7961)
[AB#9496](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/9496)

### CODE REVIEW

File: get-test-results.sh [hunk 1]
```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+# Default values
+timeout=600
+resultsDir="/app/allure-results"
+resultFilePrefixes=("mp-authority" "mp-supplier" "pb")
```
Comment (talawson05) [inline get-test-results.sh:6]: nightmare, we'll need to keep adding to this as we add new test packs. Can we not scoop this info somehow?

**Comment by bjss-chris** [comment]
@gavinwakeley1, @PhysicsJon – can we close this PR and delete the branch?

**Comment by gavinwakeley1** [comment]
Deffo for me - I think we've worked around this requirement anyway

**Comment by bjss-chris** [comment]
> Deffo for me - I think we've worked around this requirement anyway

Thanks @gavinwakeley1. I'll wait for @PhysicsJon's thoughts before I start closing and deleting stuff!

**Comment by PhysicsJon** [comment]
I agree, we have a better method of test retrieval now anyway

**Comment by bjss-chris** [comment]
> I agree, we have a better method of test retrieval now anyway

Thanks @PhysicsJon. I'll close the PR and delete the branch.

**Pull request closed without merging**